packages/js/src/generators/release-version/utils/package-manager-install.ts

```diff
@@ -0,0 +1,128 @@
+import { detectPackageManager, output } from '@nx/devkit';
+import { exec } from 'child_process';
+import { gte } from 'semver';
+
+export async function packageManagerInstall({
```

Comment:
Did the existing `installPackagesTask()` not work? If not we should probably look to enhance that one rather than create a fresh one

The dry run element can live outside of this as well (so that it aligns better with `installPackagesTask`), the returned function from the generator could apply that check and call `installPackagesTask` within its implementation

```ts
installCallback: () => {
  if (!dryRun) {
    installPackagesTask();
  }
 // ...etc...
}
```

Reply:
The existing `installPackagesTask()` would work if I heavily modified that function to take additional arguments. I don't think we should do that, since updating the lock file is a different task than just installing packages. I think we should keep a separate `updateLockFile` function for now and move it to @nx/devkit if it's ever needed elsewhere outside of `nx release`.